server/plugin/virtual_agent_test.go

```diff
@@ -335,7 +335,7 @@ func Test_CreateDefaultDateAttachment(t *testing.T) {
 				Text: "mockLabel",
 				Actions: []*model.PostAction{
 					{
-						Name: "Select mockUIType",
+						Name: "Set mockUIType",
```

Comment:
does the same text also present on the button which opens the respective modal?

Reply:
Yes, Fixed now